### PR TITLE
[TRAFODION-2389] Update some messages in the range 1000-1059

### PR DESCRIPTION
**Pull request by DaveBirdsall**
This is the first part of the changes needed for JIRA TRAFODION-2389.

In this set of changes:
1. The following messages were added to the Messages Guide: 1018, 1032, 1034, 1052 and 1055.
2. The following messages were updated in the Messages Guide: 1004, 1013, 1027, 1030, 1036, 1037, 1039, 1048 and 1058.
3. The following messages were updated in SqlciErrors.txt (note none of these occur in any existing regress tests): 1020, 1034 and 1039.
4. The following messages were removed as they are obsolete: 1019, 1038, 1054, 1056 and 1057.

### CODE REVIEW

**Comment by DaveBirdsall** [comment]
@robertamarton, if you could review the security-related messages I'd really appreciate it. I wasn't completely sure of what to write for Recovery on some of them.

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 1]
```diff
@@ -254,6 +254,24 @@ local super ID can perform a replication operation for a catalog and for
...
 
+[[SQL-1018]]
+== SQL 1018
+
+```
+Grant of role or privilege <privilege> from <user-or-role-1> to <user-or-role-2> not found, revoke request ignored.
+```
+
+Where <privilege> is a {project-name} privilege and <user-or-role-1> and <user-or-role-2> are user IDs or roles.
+
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: user-or-role-2 can be the special role PUBLIC but user-or_role-1 cannot be PUBLIC
You might want to call the <authorization IDs> like other messages.

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 2]
```diff
@@ -761,40 +814,42 @@ constraint specified in the CREATE INDEX statement.
...
 ```
-{project-name} objects cannot be created on <volume-name>: File system error <error-number>.
+Object <privilege-object> already exists.
 ```
 
-Where <volume-name> is the name of a volume.
+Where <privilege-object> is an object you specified on a REGISTER statement.
 
-Where <error-number> is the file system error number.
+*Cause:* You attempted to register a new privilege object in the
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: Actually, a <privilege-object> is a component object name.  So perhaps:
"Where <privilege-object> is a component object name you specified for a REGISTER COMPONENT statement."
"Cause: You attempted to register a new component in the {project-name} database but this component object has already been registered."
...
"Recovery:  If the component object name is incorrect, correct and resubmit."

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 3]
```diff
@@ -486,62 +534,52 @@ catalog name and resubmit.
...
 
-*Cause:* You attempted to drop SQL while one or more user-created
-catalogs existed.
+*Cause:* You attempted to revoke a privilege from <auth-id-1>, but 
+that user or role has granted that privilege to a third authorization ID.
 
 *Effect:* The operation fails.
 
-*Recovery:* You must drop all user-created catalogs before dropping SQL.
+*Recovery:* None.
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: Recovery: Reconnect to the {project-name} database using an authorization ID
that can revoke privileges from <auth-id-2> that were granted by <auth-id-1> and resubmit.

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 4]
```diff
@@ -486,62 +534,52 @@ catalog name and resubmit.
...
 == SQL 1039
 
 ```
-The DROP SQL statement could not be executed.
+No privileges were revoked. You lack the grant option for the specified privilege(s).
 ```
 
-*Cause:* See the accompanying error message for the cause.
+*Cause:* You attempted to revoke privileges from an authorization ID
+that does not have the grant option for those privileges.
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: Cause: you attempted to revoke privileges from an authorization ID but you do not have ability (grant option) to perform this operation.

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 5]
```diff
@@ -761,40 +814,42 @@ constraint specified in the CREATE INDEX statement.
...
 
-Where <lock-name> is the name of the DDL lock.
+Where <table-name> is the name of the histogram table that could not be created.
 
-*Cause:* The statement required creation of a lock with a unique lock
-name, which was not generated.
+*Cause:* You attempted to execute a CREATE SCHEMA statement, but one
+of the histogram tables needed for the schema could not be created.
+This message may be accompanied by other messages that shed more light
+on the problem.
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: Consistency:  in a previous message (1034) you stated "Additional error messages may give insight to the cause. " Here you state "This message may be accompanied by other messages that shed more light on the problem." Probably should say the same thing ...

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc (hunk 1 above)
Comment (DaveBirdsall) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: I didn't see PUBLIC mentioned anywhere in the MESSAGES guide. But I've tried some wordsmithing which you'll see in the next draft. Let me know (then) if you think it is appropriate.

**Comment by DaveBirdsall** [comment]
@robertamarton, I've made updates based on your comments. Could you take another look? Thanks.

**Comment by robertamarton** [comment]
Looks good

**Pull request merged**